doc/1/controllers/document/search/index.md

```diff
@@ -0,0 +1,136 @@
+      index: String,
+      collection: String,
+      searchQuery: ConcurrentHashMap<String?, Any?>?,
+      scroll: String? = null,
+      from: Int = 0,
+      size: Int? = null): CompletableFuture<SearchResult>
+```
+ 
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                            | Index                             |
+| `collection`       | <pre>String</pre>                            | Collection                        |
+| `searchQuery`      | <pre>ConcurrentHashMap</pre>                 | Search query                      |
+| `options`          | <pre>SearchOptions</pre>                     | Query options                     |
+---
+
+### searchQuery body properties:
+
+- `query`: the search query itself, using the [ElasticSearch Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/query-dsl.html) syntax.
+- `aggregations`: control how the search results should be [aggregated](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/search-aggregations.html)
+- `sort`: contains a list of fields, used to [sort search results](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/search-request-sort.html), in order of importance.
+
+An empty body matches all documents in the queried collection.
+
+### options
+
+A [SearchOptions](/sdk/java/3/core-classes/search-options) object.
+
+The following options can be set:
+
+| Options    | Type<br/>(default)              | Description                                                                                                                                                                                                       |
+| ---------- | ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `from`     | <pre>Integer</pre><br/>(`0`)    | Offset of the first document to fetch                                                                                                                                                                             |
+| `size`     | <pre>Integer</pre><br/>(`10`)   | Maximum number of documents to retrieve per page                                                                                                                                                                  |
+| `scroll`   | <pre>String</pre><br/>(`""`)    | When set, gets a forward-only cursor having its ttl set to the given value (ie `1s`; cf [elasticsearch time limits](https://www.elastic.co/guide/en/elasticsearch/reference/7.3/common-options.html#time-units)) |
+
+## Return
+
+Returns a [SearchResult](/sdk/java/3/core-classes/search-result) object.
```

Comment:
Write the documentation of searchResult in the repo and link it from here (/sdk/jvm/1/core-classes/search-result)